### ISSUE TITLE
AMD GPU Ubuntu 22.04 munmap_chunk(): invalid pointer

### ISSUE THREAD

**Issue by Era-Dorta** [LizardByte/Sunshine#1363]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

Sunshine crashes on launch on AMD GPU Ubuntu 22.04. I also tried with X11 and I got almost the same error. I have Windows on dual boot and sunshine runs fine on it.

### Host Operating System

Linux

### Operating System Version

Ubuntu 22.04

### Architecture

64 bit

### Sunshine commit or version

0.20.0.8ff2022aaf7c4d19b57f505df9b7d7e1b6060365

### Package

Linux - deb

### GPU Type

AMD

### GPU Model

Radeon RX 6700 XT

### GPU Driver/Mesa Version

4.6.0

### Capture Method (Linux Only)

KMS

### Config

```Shell
`~/.config/sunshine/sunshine.conf` exists but it is empty
```

### Relevant log output

```shell
[2023:06:11:19:24:32]: Info: Sunshine version: 0.20.0.8ff2022aaf7c4d19b57f505df9b7d7e1b6060365
Cannot load libcuda.so.1
[2023:06:11:19:24:32]: Error: Couldn't load cuda: -1
[2023:06:11:19:24:32]: Info: Found display [wayland-0]
[2023:06:11:19:24:32]: Info: Found interface: wl_output(4) version 2
[2023:06:11:19:24:32]: Info: Found interface: zxdg_output_manager_v1(5) version 3
[2023:06:11:19:24:32]: Warning: Missing Wayland wire for wlr-export-dmabuf
[2023:06:11:19:24:32]: Info: Found display [wayland-0]
[2023:06:11:19:24:32]: Info: Found display [wayland-0]
[2023:06:11:19:24:32]: Info: Found interface: wl_output(4) version 2
[2023:06:11:19:24:32]: Info: Found interface: zxdg_output_manager_v1(5) version 3
[2023:06:11:19:24:32]: Info: Offset: 0x0
[2023:06:11:19:24:32]: Info: Resolution: 3840x2160
[2023:06:11:19:24:32]: Info: Name: HDMI-1
[2023:06:11:19:24:32]: Info: Found monitor: LG Electronics 27"
[2023:06:11:19:24:32]: Info: HDMI-1: LG Electronics 27"
[2023:06:11:19:24:32]: Info: // Testing for available encoders, this may generate errors. You can safely ignore those errors. //
[2023:06:11:19:24:32]: Info: Trying encoder [nvenc]
[2023:06:11:19:24:32]: Info: Screencasting with KMS
[2023:06:11:19:24:32]: Info: Found monitor for DRM screencasting
[2023:06:11:19:24:32]: Error: Missing extension: [EGL_EXT_image_dma_buf_import_modifiers]
munmap_chunk(): invalid pointer
Aborted (core dumped)
```

**Comment by Era-Dorta** [comment]
Bump to avoid closing.

**Comment by Era-Dorta** [comment]
Solved with version 0.21.0, for all I know it might have been solved by updates in the OS instead of updates in the sunshine.
